src/Raven.Server/ServerWide/ServerStatistics.cs

```diff
@@ -10,8 +12,31 @@ public ServerStatistics()
             StartUpTime = SystemTime.UtcNow;
         }
 
+        public TimeSpan UpTime => SystemTime.UtcNow - StartUpTime;
+
         public readonly DateTime StartUpTime;
-        
+
         public DateTime? LastRequestTime { get; set; }
+
+        public void WriteTo(BlittableJsonTextWriter writer)
+        {
+            writer.WriteStartObject();
+
+            writer.WritePropertyName(nameof(UpTime));
```

Comment:
Do we want to expose this in different format? E.g. in seconds only (separate property)? Note that monitoring should be done via SNMP anyway IMO.